tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/ZeroRetryJitterTest.java

```diff
@@ -62,6 +62,7 @@ public void test() {
             assertTrue("It took too much time for 3 retries", zeroJitterClient.getTotalRetryTime() < 3 * 200);
         }
         catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Don't print the stack trace like this.

Either don't catch the exception (and allow it to be thrown from the test method), or include it as the second argument to `fail()`. Both of these methods should ensure that the exception is included in the test failure report, without having to dig into the server logs to find the stack trace.